src/main/java/org/sagebionetworks/template/cdn/CdnBuilderImpl.java

```diff
@@ -60,6 +61,8 @@ VelocityContext createContext() {
 		// The ACM ARN is the same as the one used for portal
 		String acmCertificateArn = config.getProperty(PROPERTY_KEY_BEANSTALK_SSL_ARN+"portal");
 		ctxt.put(CTXT_KEY_ACM_CERT_ARN, acmCertificateArn);
+		String stackInstanceAlias = config.getProperty(PROPERTY_KEY_STACK_INSTANCE_ALIAS);
```

Comment:
This function should take in input the type and put the needed properties accordingly